docs/content/quickstart.md

```diff
@@ -3,116 +3,127 @@ title: QuickStart Guide
 
-The `porter build` command will generate the bundle:
+## Run the install instructions from a bundle located in a registry
+
+To install a bundle, you use the `porter install` command. 
 
 ```
-porter build
+porter install --tag getporter/porter-hello:v0.1.0
+
 ```
 
-## Install the bundle
+In this example, you are installing athe 0.1.0 version of the bundle from the default registry by specifying a tag. 
+
+## List 
 
-You can then use `porter install` to install your bundle:
+To see the bundle installations, use the `porter list` command. 
 
 ```
-porter install
+porter list
 ```
 
-If you wish to uninstall the bundle, you can use `porter uninstall`:
-
+Sample Output:
 ```
-porter uninstall
+NAME       CREATED          MODIFIED         LAST ACTION   LAST STATUS
+HELLO      21 minutes ago   21 minutes ago   install       succeeded
 ```
 
-## Publish the bundle
+In this example, it shows the bundle metadata along with the creation time, modification time, the last action that was performed, and the status of the last action. The default name of the bundle, HELLO, is used since there was no installation name specified at the command line. 
 
-When you are ready to share your bundle, the next step is publishing it to an
-OCI registry such as Docker Hub or Quay.
+## Show
 
-You must authenticate with `docker login` before publishing the bundle. Make
-sure that the `tag` listed in your `porter.yaml` is a reference to which the
-currently logged in user has write permission.
+To see information about an installation, use the `porter show` command with the name of the installation.
 
-```yaml
-tag: myregistry/porter-hello:v0.1.0
 ```
-
-Now run `porter publish` and porter will push the invocation image and bundle to
-the locations specified in the **porter.yaml** file:
-
+porter show HELLO
 ```
-porter publish
+
+Sample Output:
 ```
+Name: HELLO
+Created: 48 minutes ago
+Modified: 36 minutes ago
+
+History:
+----------------------------------------
+  Action     Timestamp       Status
+----------------------------------------
+  install    48 minutes ago  succeeded
+  uninstall  42 minutes ago  succeeded
+  install    36 minutes ago  succeeded
+  ```
 
-## Install from the registry
 
-Now that your bundle is in a registry, anyone can use a [CNAB-compliant
-tool][tools], not just Porter, to install the bundle. 
+## Run the upgrade instructions from the bundle
 
-Previously when we use
-`porter install` when we were in the same directory as a porter bundle, we
-didn't specify an installation name to create, so Porter defaulted the
-installation to the name of the bundle. This time we will explicitly name the
-installation "demo".
+To run the upgrade instructions from the bundle, use `porter upgrade`.
 
 ```
-porter install demo --tag getporter/porter-hello:v0.1.0
+porter upgrade 
```

Comment:
This command is missing the name of the installation. I assume you ran this in a directory with a porter.yaml but that's not what the quickstart is having the reader do to get to this point.

```suggestion
porter upgrade HELLO
```